docs/import-old-pulses.md

```
@@ -0,0 +1,62 @@
+# Submissions CSV Import Guide
+
+## Required columns:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)